### PR TITLE
Properly display the lessons on the sidebar

### PR DESCRIPTION
**Pull request by Kleidukos**
image

Also:

* Use JetBrains Mono to display code
* Remove ^M character from .scss files

### CODE REVIEW

**Comment by JonathanLorimer** [comment]
@Kleidukos A lot of these diffs seem to be whitespace changes. Could you go through and self review, and comment on the areas that need attention, it is difficult for me to try and diff each file with my eyes and spot the change lol.

File: _includes/section-home-helloworld.html [hunk 1]
```diff
@@ -1,18 +1,22 @@
 {% capture getting_started %}
```
Comment (Kleidukos) [inline _includes/section-home-helloworld.html:1]: @JonathanLorimer This file

File: _includes/section-sidebar-menu.html [hunk 2]
```diff
@@ -1,14 +1,15 @@
```
Comment (Kleidukos) [inline _includes/section-sidebar-menu.html:1]: @JonathanLorimer This file as well

File: _includes/_head.html [hunk 3]
```diff
@@ -51,12 +51,6 @@
...
-  <script type="text/javascript">
-  !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t){var e=document.createElement("script");e.type="text/javascript";e.async=!0;e.src=("https:"===document.location.protocol?"https://":"http://")+"cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(e,n)};analytics.SNIPPET_VERSION="3.1.0";
-  analytics.load("C0p07Sw9j7mZnycKDCgPOAyqJR5hhyoQ");
-  analytics.page()
-  }}();
-  </script>
```
Comment (Kleidukos) [inline _includes/_head.html:59]: Removing Google Analytics

File: _data/contents.yml [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
   - home
   - report
+basics:
+  - basics
+  - functions
+  - types
+  - algebraic-data-types
+  - lists-tuples
+  - pattern-matching-control-flow
+  - containers
```
Comment (Kleidukos) [inline _data/contents.yml:11]: Lesson plan

File: _config.yml [hunk 5]
```diff
@@ -33,6 +33,9 @@ assets:
     - _assets/stylesheets
     - _assets/fonts
 
+
+ghc_version: 9.0
+
```
Comment (Kleidukos) [inline _config.yml:38]: Version of GHC we're using

Comment (JonathanLorimer) [reply _config.yml:38]: nice

**Review by JonathanLorimer — APPROVED** [review]

**Pull request merged**